NeosModLoader/ModLoaderConfiguration.cs

```diff
@@ -86,25 +126,39 @@ internal static ModLoaderConfiguration Get()
 					}
 				}
 			}
-			return _configuration;
+
+			try
+			{
+				var boolType = typeof(bool);
+				foreach (var option in configOptions)
+				{
+					if (LaunchArguments.TryGetArgument($"Config.{option.Name}", out var argument))
+					{
+						if (option.PropertyType == boolType)
+						{
+							option.SetValue(config, true);
+							Logger.MsgInternal($"Enabling [{option.Name}] from launch flag");
+						}
+						else if (!argument.IsFlag)
+						{
+							config.SetProperty(option, argument.Value!);
+							Logger.MsgInternal($"Setting [{option.Name}] from launch flag: {argument.Value}");
+						}
+					}
+				}
+			}
+			catch
+			{
+				throw;
+			}
```

Comment:
.... is there a point to a try catch that just throws anyways?

Reply:
oh yea, that's a derp lol